src/test/java/com/cloudbees/hudson/plugins/folder/FolderTest.java

```diff
@@ -451,6 +451,14 @@ private void copyViaHttp(Folder f, JenkinsRule.WebClient wc, String fromName, St
         assertFalse(f.hasVisibleItems());
     }
 
+    @Test public void getItemsPredicate() throws IOException {
+        final Folder d = r.jenkins.createProject(Folder.class, "d");
+        final FreeStyleProject p1 = d.createProject(FreeStyleProject.class, "p1");
+        FreeStyleProject p2 = d.createProject(FreeStyleProject.class, "p2");
+        FreeStyleProject c1 = d.createProject(FreeStyleProject.class, "c1");
```

Comment:
just to keep them all the same
```suggestion
        final FreeStyleProject p1 = d.createProject(FreeStyleProject.class, "p1");
        final FreeStyleProject p2 = d.createProject(FreeStyleProject.class, "p2");
        final FreeStyleProject c1 = d.createProject(FreeStyleProject.class, "c1");
```